06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * The Inverted Index class.
+ * @class
+ */
+class InvertedIndex {
+
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.unique = [];
+    this.tableObj = {};
```

Comment:
Variable Naming: I'm sure I've stressed on not using datatypes in variable names. `Obj` is `Object` no?